Documentation/tutorial/intro_toc.md

```diff
@@ -40,14 +40,14 @@ Three kinds of links are supported:
 ```yml
 - name:
   href:
-  homepage:
+  topicHref:
 - name:
   href:
-  homepage:
+  topicHref:
   items:
     - name:
       href:
```

Comment:
Yes, href remains the same usage, which can be file/folder/toc.

---
In reply to: [73336120](https://github.com/dotnet/docfx/pull/537#discussion_r73336120) [](ancestors = 73336120)